articles/open-source-licencses/index.md

```diff
@@ -0,0 +1,108 @@
+
+- Commercial Use of Software
+- Modification
+- Distribution
+- Private Use of Software
+  
+#### Limitations
+- No warranty or liability is provided
+  
+
+### GNU General Public License(GPL)
+
+There are two versions of the GNU GPL license. The earlier version is called GPL 2.0, and the latter version is called GPL version 3.0. It is a good practice to include the latest version of any license. Let us discuss the GNU GPL 3.0 license in detail.
+
+#### Permissions
+- Modification
+- Private Use
+  
+#### Limitations
+- Should not be commercialized
+- No private use: should be kept open-sourced.
+
+### Apache License
+
+Designed to enable faster progress of open source development, the Apache license is the go-to license for organizations releasing open-source software. The license gives complete access to the person using it, including the ability to file for a patent. This is one of the few licenses available that allows the work built using this software to be eligible for patent application.  
+
+#### Permissions
+-  Commercial use
+-  Distribution
+-  Modification
+-  Patent use
+-  Private Use
+
+#### Limitations
+
+- No warranty or liability is provided
+
+### Creative Commons Attribution
+
+This is one of the stricter licenses made for sharing knowledge with the world without boundaries. This is the sole objective of this license. Apart from this objective, it does not allow one to create, distribute copies, either original or modified, through any other channels than the original one. There are several variants of this license. 
+
+#### Permissions
+
+Share — copy and redistribute the material in any medium or format
+Adapt — remix, transform, and build upon the material
+for any purpose, even commercially.
+under the terms that attribution where necessary is given. 
+
+Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)
+
+Share — copy and redistribute the material in any medium or format
+Adapt — remix, transform, and build upon the material
+for any purpose, even commercially.
+
+under the terms that attribution where necessary is given. and the same file is shared under the same license as the original.
+
+### The Unlicense
+
+> Anyone is free to copy, modify, publish, use, compile, sell, or distribute this software, either in source code form or as a compiled binary, for any purpose, commercial or non-commercial, and by any means.
+
+#### Permissions
+
+-  Commercial use
+-  Distribution
+-  Modification
+-  Private Use
+
+
+#### Limitations
+- No warranty or liability is provided
+
+
+### Codebase with no license included
+
+Many times the author or contributors may release the code without any license file attached with it. In such cases, one should contact the contributors and ask them to include a license. This would ensure minimal legal conflicts, if any, in the future. Until a license is included, the author or contributors have exclusive rights over the code. One may not copy or use it without their permission. 
+
```

Comment:
Great article, @lalith1403! One suggestion that would make this article better would be the addition of a summary table, where you compare the permissions, limitations and "ideal for" column, that discusses the licenses in a crisp manner.